src/Microsoft.ML.Data/Evaluators/QuantileRegressionEvaluator.cs

```diff
@@ -446,12 +446,12 @@ private void CheckInputColumnTypes(DataViewSchema schema)
 
             var t = schema[(int)LabelIndex].Type;
             if (t != NumberDataViewType.Single)
-                throw Host.ExceptSchemaMismatch(nameof(schema), "label", LabelCol, "float", t.ToString());
+                throw Host.ExceptSchemaMismatch(nameof(schema), "label", LabelCol, "Single", t.ToString());
 
             VectorType scoreType = schema[ScoreIndex].Type as VectorType;
             if (scoreType == null || scoreType.Size == 0 || (scoreType.ItemType != NumberDataViewType.Single && scoreType.ItemType != NumberDataViewType.Double))
             {
-                throw Host.ExceptSchemaMismatch(nameof(schema), "score", ScoreCol, "known-size vector of float or double", t.ToString());
+                throw Host.ExceptSchemaMismatch(nameof(schema), "score", ScoreCol, "known-size vector of Single or Double", t.ToString());
```

Comment:
>vector [](start = 95, length = 6)

would array be more correct? vector still needs some mapping to a C# type.

Reply:
My problem with array, is what we also can work with `VBuffer` in our code.
Would like to hear other people opinion as well, tho.

---
In reply to: [268224361](https://github.com/dotnet/machinelearning/pull/3046#discussion_r268224361) [](ancestors = 268224361)